### ISSUE TITLE
Include an announcement banner to inform users about the default redirection being set to the 'version/dev' path

### ISSUE THREAD

**Issue by Revathyvenugopal162** [ansys/pyansys#423]
### Description of the modifications

The banner conveys information regarding the default redirect path, now set to 'version/dev.'  the banner can be reuse from pydata or we can write own own banner.

### Useful links and references

https://pydata-sphinx-theme.readthedocs.io/en/latest/index.html

**Comment by RobPasMue** [comment]
We need to clarify what is it we want to inform users about with this banner - because I still don't get it.

**Comment by RobPasMue** [comment]
We want to inform them that they are being redirected to the "dev" version? Most people expect these docs to be a general landing page rather than a Python package landing page. Following that thought... I think the banner would introduce more confusion rather than clarity.

**Comment by Revathyvenugopal162** [comment]
> We want to inform them that they are being redirected to the "dev" version? Most people expect these docs to be a general landing page rather than a Python package landing page. Following that thought... I think the banner would introduce more confusion rather than clarity.

i mean the other way as well, Users are required to manually identify their installed pyansys version from a dropdown menu rather than having a direct match with the landing page version of corresponding library or they have to  upgrade their library.

**Comment by RobPasMue** [comment]
I think I am not following you @Revathyvenugopal162...

Do you want to inform users that have installed the ``pyansys`` metapackage, that they should switch to their specific installed version (using the switcher) to have access to the docs of the dependencies (i.e. different PyAnsys packages) installed?

**Comment by Revathyvenugopal162** [comment]
> I think I am not following you @Revathyvenugopal162...
> 
> Do you want to inform users that have installed the `pyansys` metapackage, that they should switch to their specific installed version (using the switcher) to have access to the docs of the dependencies (i.e. different PyAnsys packages) installed?

Yeah something similar. it is a just a suggestion  If you find it unrelated, we can disregard.

**Comment by RobPasMue** [comment]
Let's do the following. Propose a PR with your suggestion. We can iterate on it 😃
